src/routes/api/collabs/[id]/picks/[pick_id]/index.ts

```diff
@@ -153,6 +199,9 @@ export const del: RequestHandler = async ({ request, params }) => {
 	const decodedUser = Jwt.decode(cookies['user_id']);
 	const userId = decodedUser['user_id'] as string;
 
+	const urlParams = new URLSearchParams(request.url.split('?')[1]);
+	const reason = urlParams.get('reason');
```

Comment:
Since we're not querying data, it's probably better to put the reason in the body of the request. This way you don't have to look for URL params either.

Reply:
I wasn't able to pass the reason through the body but instead, I passed it through headers.